content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -683,6 +687,62 @@ More configuration options can be found on the [kong-plugin-ddtrace][3] plugin d
 [3]: https://github.com/DataDog/kong-plugin-ddtrace#configuration
 
 {{% /tab %}}
+
+{{% tab "Apache HTTPd" %}}
```

Comment:
[Minor, non-blocking] I searched around the [website](https://httpd.apache.org/) a bit and I only found either _Apache HTTP Server_ or _Apache httpd_, but never _Apache HTTPd_. Not against it, but I was just curious if that was intentional.

Reply:
I also search because I've seen _HTTPd_, _Apache HTTP Server_, _Apache httpd_, and _Apache HTTPd_. After more research, I think it's a common mistake related to [HTTPd](https://en.wikipedia.org/wiki/Httpd). 

Now, I shall rename the section _Apache HTTP Server_. It's a bit verbose but very explicit.

Reply:
```suggestion
{{% tab "Apache HTTP Server" %}}
```